### DISCUSSION TITLE
反応しないキーがある

### DISCUSSION THREAD

**Discussion by kbyamy** [Yowkees/keyball#227] [Q&A] [answered]
Keyball 61を組み立てています。

ビルドガイドの順に作業を進め、正規ファームウェアを書き込んでREMAPのTest Matrixを使いキーの反応を確かめてみたところ1箇所だけ反応しないキーがありました。

![PXL_20230506_133301510](https://user-images.githubusercontent.com/43465759/236631100-9ebd7d9a-3e92-4818-bcd2-a8b8129c559b.jpg)

以下の内容を試してみましたが、解決の糸口がわからず...なにかアドバイスをいただけないでしょうか。よろしくお願いします。
- スイッチを外してピンセットでショートさせてみても反応せず
- 周辺のダイオードの向きを確認→問題なし
- 他のキーはすべて反応する
- トラックボールも反応する

**Comment by Yowkees** [comment, marked as answer]
![KB61R_fix](https://github.com/Yowkees/keyball/assets/84209772/7004053a-8630-4eea-90c9-1eae66568518)

ご連絡が遅くなり申し訳ありませんでした。

ご連絡いただいたキーに対応するダイオードは緑丸で示した箇所になります。
写真からははっきりとは判断できませんが、左側の端子が半田付けされていないように見えますのでご確認いただけますでしょうか。
それか、緑丸ダイオードがテンプラしていないかご確認お願いいたします。

**Reply by kbyamy** [reply]
お忙しい中ご回答ありがとうございます。

ご指摘いただいたダイオードのはんだ付けをやりなおしてみたところ、反応しなかったキーが無事に反応するようになりました。
的確なサポートに感謝します！

**Comment by Yowkees** [comment]
無事に動作したようで良かったです！安心しました。
ぜひ末永くご愛用いただければ幸いです。
